pages.id/linux/a2ensite.md

```diff
@@ -0,0 +1,12 @@
+# a2ensite
+
+> Izinkan sebuah host maya Apache pada OS berbasis Debian.
```

Comment:
```suggestion
> Izinkan sebuah host maya (virtual host) Apache pada OS berbasis Debian.
```

The technical name for this is `VirtualHost`, and adding an English translation here could help some people who aren’t familiar with standard Indonesian words such as `awakutu` for `debugging`.